model-api/src/commonMain/kotlin/org/modelix/model/api/ITree.kt

```diff
@@ -254,6 +254,12 @@ interface ITree {
      */
     fun deleteNodes(nodeIds: LongArray): ITree
 
+    /**
+     * @param nodeId id of the node
+     * @param concept the concept ref of the new concept for the node, or null to remove the concept
+     */
+    fun setConcept(nodeId: Long, concept: IConceptReference?): ITree
```

Comment:
Oh. I meant to add this comment to `INode.replaceNode(concept: ConceptReference): INode`.
There `concept` could be nullable.

Reply:
I didn't make it nullable since in MPS the concept is mandatory. One could map a `null` value to `BaseConcept` in that case but that might be confusing. I think the user should specify `BaseConcept` explicitly.

Reply:
Hm. But `addNewChild` etc. has also nullable concepts...
Let's be consistent.